### ISSUE TITLE
Issues with i18n

### ISSUE THREAD

**Issue by Gozer-404** [mintty/mintty#1200]
Several issues after the latest review of the fr.po file

1.  Some strings in pull-down menus are not translated
![image](https://user-images.githubusercontent.com/18137218/215039808-39bd01f9-5d79-47c1-9527-02bc8d097dbc.png)
![image](https://user-images.githubusercontent.com/18137218/215039839-ffa362da-2c66-4e98-ac06-b7e408a061c2.png)

2. One string is too big (and BTW I don't understand what it is supposed to configure) :   "Show size while selecting (0..12)"
![image](https://user-images.githubusercontent.com/18137218/215040117-e7858307-46f2-4030-86da-a956c0056214.png)

3. Need some understanding of the following sentences, I don't get what it really means, thus difficult to translate:
```
#: config.c:2288
msgid "@ Windows language @"   <= probably current language set in Windows ?
msgstr "@ Langue système @"

#: config.c:2289
msgid "* Locale environm. *"   <= what is the difference with this one ?
msgstr "* Locale environ. *"

#: config.c:2290
msgid "= cfg. Text Locale ="
msgstr "= Langue caractères ="  <= same question here

#. __ Options - Terminal:
#: config.c:4435
msgid "&Answerback"  <= did not get what it means
msgstr "&Retour"
```

**Comment by BrianInglis** [comment]
```
#. __ Options - Terminal:
#: config.c:4435
msgid "&Answerback"  <= did not get what it means
msgstr "&Retour"
```
Answer back response string when `ENQ` sent to terminal: `"&Réponse"`?

**Comment by mintty** [comment]
1. Some pull-down settings are shared among multiple options, e.g. "full"/"complet" appears both in FontSmooting and in EmojiPlacement (see first picture). The problem with translating them is that they are also used as settings on the command line or in the config file. To keep this consistent they should rather not be translated at all and the fact that EmojiPlacement has localizable settings is a problem and probably a mistake, to be checked.
2. This is option SelectionShowSize, which causes an indicator for text selection, presenting the current size while selecting. Values can choose the position of the popup indicator, maybe a bit overdrawn.
3. Option Language lets you choose how to derive the mintty UI language if not selected explicitly. It may be taken from the Windows desktop language (@) or from the cygwin locale setting when mintty was started (*) or from an explicitly configured locale (Options section "text"); there is a subtle difference between the latter two but I thought it would make sense to leave users the choice, maybe also a bit overdrawn.
Answerback is a historic terminal feature. When sending a Ctrl+E control character, they would respond their individual identification. Thereby, in mainframe times, the system could determine which exact terminal a user was sitting at. Brian's suggestion réponse sounds good.

**Comment by mintty** [comment]
Committed an enhancing tweak.

**Comment by mintty** [comment]
The enhancement was released with 3.6.4, hopefully sufficient, so closing this issue.
